frontend/src/locale/index.ts

```diff
@@ -120,6 +120,60 @@ Let's get in touch to find the best solution for your business' needs!`,
+    "profile.myGrant.status.approved": "Approved",
+    "profile.myGrant.status.approved.nextSteps":
+      "Your grant request is approved. We will send you an email very soon with further instructions. Check your email regularly.",
+    "profile.myGrant.status.waiting_list": "Waiting List",
+    "profile.myGrant.status.waiting_list.nextSteps":
+      "You are on the waiting list. We will notify you as soon as we can accept your grant.",
+    "profile.myGrant.status.waiting_list_maybe": "Waiting List",
+    "profile.myGrant.status.waiting_list_maybe.nextSteps":
+      "You are on the waiting list. We will notify you as soon as we can accept your grant.",
+    "profile.myGrant.status.waiting_for_confirmation":
+      "Waiting for Confirmation",
+    "profile.myGrant.status.waiting_for_confirmation.nextSteps": `Congratulations, your grant request has been approved!
+
+Use the 'Manage' button on the page to confirm or decline the grant. You have until {replyDeadline} to confirm or decline the grant, after which your grant will be automatically revoked.
+`,
+    "profile.myGrant.status.refused": "Refused",
+    "profile.myGrant.status.refused.nextSteps":
+      "You have declined the grant offer. If this was a mistake, please contact us immediately.",
+    "profile.myGrant.status.confirmed": "Confirmed",
+    "profile.myGrant.status.confirmed.nextSteps":
+      "Thank you for confirming your grant. We look forward to seeing you at the conference! We will send you the voucher for the ticket soon, please check your email regularly.",
+    "profile.myGrant.status.did_not_attend": "Did Not Attend",
+    "profile.myGrant.status.did_not_attend.nextSteps":
+      "You did not attend the conference.",
+    "profile.myGrant.edit": "Edit",
+    "profile.myGrant.editInfo":
+      "Ensure all the information provided are correct. You have until {editDeadline} to edit your information.",
+    "profile.myGrant.manage": "Manage",
+    "profile.myGrant.manage.warning": `If your plans change or you cannot attend the conference anymore, please let us know by clicking the 'Manage' button to refuse your grant or email us at {grantsEmail} as soon as possible.
+Failing to notify us may impact your eligibility for financial aid at future events.`,
+    "profile.myGrant.noGrant.heading": "You haven't requested a grant",
+    "profile.myGrant.noGrant.body.canSubmit": `If you're facing financial difficulties and wish to attend PyCon Italia, our grant application form is currently open!
+Find our more information {grantInfoLink} and submit your application below.`,
+    "profile.myGrant.noGrant.submitGrant": "Request a grant",
+    "profile.myGrant.noGrant.body.closed":
+      "The grant application form is currently closed. Stay tuned for future opportunities.",
+    "profile.myGrant.noGrant.body.openingSoon":
+      "Our grant application form will open soon! Check our {grantInfoLink} for more details.",
```

Comment:
why we have a link in the text here?

Reply:
why not? to the info page, Alessia suggested

Reply:
The button below already links to it?

Reply:
ah I didn't know

Reply:
reverted this